deps/rabbit/src/rabbit_header_routing_node_interceptor.erl

```diff
@@ -0,0 +1,17 @@
+-module(rabbit_header_routing_node_interceptor).
```

Comment:
Can we rename this module to `rabbit_message_interceptor_routing_node`?
This way all `rabbit_message_interceptor*` modules show up next to each other in the file menu.